src/ListViewSwipe.ts

```diff
@@ -234,6 +245,7 @@ class ListViewSwipe extends WidgetBase {
         const guid = registry.byNode(element).getGuid();
         const context = this.createContext(guid);
         this.callMicroflow(direction, context);
+        this.callNanoflow(direction, context);

```

Comment:
I think this can be simplified by creating one function that handles microflow, page and nanoflow coz all the 3 functions take in the same number of parameters.

Reply:
The have different API